towncrier.toml

```diff
@@ -0,0 +1,50 @@
+[tool.towncrier]
+version = "2.15.0-dev0"
+directory = "doc/whatsnew/fragments"
+filename = "doc/whatsnew/2/2.15/index.rst"
```

Comment:
The version and filename should be updated through a `tbump` script later on.

Reply:
Search for `__pkginfo__.py` in the tbump conf, to see how it's done :)

Reply:
Thanks for the hint! Just replacing the `version` is the easy part. Changing the `filename` would require a simple helper script which restricts the version to MAJOR.MINOR.

Reply:
How does this work with fragments of older versions then? Won't they get mixed up?